src/pfe/portal/routes/templates.route.js

```diff
@@ -47,7 +45,7 @@ router.get('/api/v1/templates', validateReq, async (req, res, _next) => {
  */
 router.get('/api/v1/templates/repositories', sendRepositories);
 
-router.post('/api/v1/templates/repositories', validateReq, (req, res, _next) => {
+router.post('/api/v1/templates/repositories', validateReq, async (req, res, _next) => {
```

Comment:
Now that this middleware function is async, you have to wrap it in a try/catch block or errors will be suppressed. We have the top-level error handler in `server.js`, but that only works for sync functions. 

Search for `You must catch errors that occur in asynchronous code` in  https://expressjs.com/en/guide/error-handling.html

This applies to a number of the functions below, too.

Reply:
good spot, I've added an async error catcher to all our routes using just 1 line (server.js), via a nifty package I found https://www.npmjs.com/package/express-async-errors. (I tested manually that it works)